apps/champions/test/battle_test.exs

```diff
@@ -680,6 +680,158 @@ defmodule Champions.Test.BattleTest do
       assert "team_1" ==
                Champions.Battle.Simulator.run_battle([unit], [target_dummy_5], maximum_steps: maximum_steps).result
     end
+
+    test "Self Heal" do
```

Comment:
This test is wrong: if I change the healing of the unit to `attack_ratio: 0` it still passes. I think the attacking unit only has time to hit once, leaving the healer at 5HP!